schedule_info.py

```diff
@@ -0,0 +1,247 @@
+from discord_slash.utils.manage_commands import create_option, create_choice
+
+week_days = ("monday", "tuesday", "wednesday", "thursday", "friday", "saturday", "sunday")  #lista dni tygodnia
+
+
+class Schedule(commands.Cog):
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.command(
+        brief="Co jest grane?",
+        help="Bot informuje o obecnie granym utworze.",
+        name='cjg'
+    )
+    @commands.cooldown(1, 60, commands.BucketType.user)
+    async def is_played(self, ctx):
+        embed = discord.Embed(title="Obecnie gramy:")
+        resp = requests.get("https://radio17.pl/api/stream/stats/extended")  # zapytanie do API o to co bedzie grane
+        json_resp = resp.json()
+        is_played = f'{json_resp["current"]["artist"]} - {json_resp["current"]["title"]}'
+        embed.add_field(name=is_played, value="https://radio17.pl")
+        await ctx.send(embed=embed)
+
+    @cog_ext.cog_slash(
+        name="cjg",
+        description="Informuje co obecnie jest grane w radiu."
+    )
+    async def _is_played(self, ctx: SlashContext):
+        embed = discord.Embed(title="Obecnie gramy:")
+        resp = requests.get("https://radio17.pl/api/stream/stats/extended")
+        json_resp = resp.json()
+        is_played = f'{json_resp["current"]["artist"]} - {json_resp["current"]["title"]}'
+        embed.add_field(name=is_played, value="https://radio17.pl")
+        await ctx.send(embed=embed)
+
+    @commands.command(
+        brief="Co będzie grane?",
+        help="Bot informuje jaki utwór będzie grany następnie.",
+        name='cbg'
+    )
+    @commands.cooldown(1, 60, commands.BucketType.user)
+    async def will_play(self, ctx):
+        embed = discord.Embed(title="Za chwilę będziemy grać:")
+        resp = requests.get("https://radio17.pl/api/stream/stats/extended")
+        json_resp = resp.json()
+        if json_resp["next"] is not None:  # jesli jest informacja o nastepnym utworze
+            will_play = f'{json_resp["next"]["artist"]} - {json_resp["next"]["title"]}'
+            embed.add_field(name=will_play, value="https://radio17.pl")
+        else:
+            err = "No nie wiem co będzie grane."
+            embed.add_field(name=err, value="https://radio17.pl")
+        await ctx.send(embed=embed)
+
+    @cog_ext.cog_slash(
+        name="cbg",
+        description="Informuje co będzie grane za chwilę w radiu."
+    )
+    async def _will_play(self, ctx: SlashContext):
+        embed = discord.Embed(title="Za chwilę będziemy grać:")
+        resp = requests.get("https://radio17.pl/api/stream/stats/extended")
+        json_resp = resp.json()
+        if json_resp["next"] is not None:
+            will_play = f'{json_resp["next"]["artist"]} - {json_resp["next"]["title"]}'
+            embed.add_field(name=will_play, value="https://radio17.pl")
+        else:
+            err = "No nie wiem co będzie grane."
+            embed.add_field(name=err, value="https://radio17.pl")

```

Comment:
Chyba można zrobić to inline, bo długości linii wynikającej z PEP8 nie przekroczy.
```suggestion
            embed.add_field(name="No nie wiem co będzie grane.", value="https://radio17.pl")

```